### DISCUSSION TITLE
Disable BossbarCompass by default (in game not config)

### DISCUSSION THREAD

**Discussion by Insprill** [Zrips/CMI#2816] [Feature Requests]
**Description of feature/improvement request:**
Have the ability to have the bossbar compass disabled by default then players can enable it if they want it
---
### Detailed explanation of what it should do, when and in what situation: ###
Have the ability to have the bossbar compass disabled by default then players can enable it if they want it
### Basic explanation as to why this feature is needed and/or useful to have: ###
Because some players might not like it and might not ask how to disable it and just assume you can't, then leave

**Comment by Va1a** [comment]
i'm pretty sure this is how it is by default... that's how it was for me.

**Comment by Insprill** [comment]
No maybe it was a little unclear how i stated it above. I want players to be able to enable it, but it be disabled by default. Not fully disabled in the config.

**Comment by katnova** [comment]
*if anyone is out there* 
I would also like to see this.
